esdk-performance-testing/benchmarks/java/pom.xml

```
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.amazon.esdk</groupId>
+    <artifactId>esdk-performance-benchmark</artifactId>
+    <version>1.0.0</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <!-- AWS Encryption SDK -->
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-encryption-sdk-java</artifactId>
+            <version>3.0.2</version>
+        </dependency>
+        
+        <!-- AWS Cryptographic Material Providers Library -->
+        <dependency>
+            <groupId>software.amazon.cryptography</groupId>
+            <artifactId>aws-cryptographic-material-providers</artifactId>
+            <version>1.0.2</version>
```

Probably blocking? The latest MPL is 1.11.0.
https://mvnrepository.com/artifact/software.amazon.cryptography/aws-cryptographic-material-providers
```suggestion
            <version>1.11.0</version>
```